CIP-1694/README.md

```diff
@@ -1339,6 +1315,69 @@ We solve the long-term participation problem by not allowing reward withdrawals
 (after the bootstrap phase) unless the stake is delegated to a DRep
 (including the two special cases, namely 'Abstain' and 'No confidence').
 
+### Changelog
+
+#### Changes post Longmont workshop (March 2023)
+
+* Thank the workshop attendees.
+* We have added Constitutional Committee term limits.
+  The term is renewed each time they successfully elect a new committee.
+* Two new "pre-defined" DRep options: abstain and no confidence.
```

Comment:
better to call pre-set, or automated

Reply:
I don't really like that name either, but that's what we currently have. If we want to change it, we should do it in a separate PR.

Reply:
thats fine, so long as "pre-defined" is the only name used across the document for these DReps

Reply:
Yes, I think that's the case